hi-ml-azure/run_pytest.py

```diff
@@ -81,6 +94,57 @@ def run_pytest(folder_to_test: str, pytest_mark: str) -> None:
         raise ValueError(f"PyTest failed with exit code: {status_code}")
 
 
+def download_run_output_file(blob_path: Path, destination: Path, run: Run) -> Path:
+    """
+    Downloads a single file from the run's default output directory: DEFAULT_AML_UPLOAD_DIR ("outputs").
```

Comment:
Maybe not a good idea to mention the value of that variable. If it changes, this docstring will be out of date.